cpp/deplex/src/deplex/cell_segment.cpp

```diff
@@ -3,39 +3,28 @@
 #include <Eigen/Eigenvalues>
 
 namespace deplex {
+CellSegment::CellSegment(Eigen::MatrixXf const& cell_points, config::Config const& config)
```

Comment:
Why do we need to store config as class member? Is it used later not in constructor?